driver-core/src/main/com/mongodb/client/model/expressions/MapExpression.java

```diff
@@ -21,45 +21,162 @@
 import static com.mongodb.client.model.expressions.Expressions.of;
 import static com.mongodb.client.model.expressions.MqlUnchecked.Unchecked.PRESENT;
 
+/**
+ * A map {@link Expression value} in the context of the MongoDB Query
+ * Language (MQL). An map is a finite, unordered a set of
+ * {@link EntryExpression entries} of a certain type, such that no entry key
+ * is repeated. It is a mapping from keys to values.
+ *
+ * @param <T> the type of the elements
```

Comment:
The map documentation does not define an "element". Should this be "entry value"?